services/json_uploader.py

```diff
@@ -10,15 +10,26 @@ def generateBin(data):
         sys.exit()
 
     print("\nGenerating bin link..")
-    url = 'https://api.jsonbin.io/v3/b'
+    
+    # if BinId is exist then update the bin otherwise create a new bin
+    BinId = os.getenv('jsonbin_bin_id')
```

Comment:
rename this variable 

https://realpython.com/python-pep8/
![image](https://user-images.githubusercontent.com/36763164/196052286-6d07fafb-dd5f-449e-9035-a4688be02833.png)

Reply:
Done, thanks for this note.